### PR TITLE
Patch: 3.2.1

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .nf-core.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ template:
   org: nf-core
   outdir: .
   skip_features: null
-  version: 3.2.0
+  version: 3.0.1
```
Comment (jfy133) [inline .nf-core.yml:21]:
```suggestion
  version: 3.2.1
```

File: conf/modules.config [hunk 2]
```diff
@@ -257,7 +257,7 @@ process {
     }
 
     withName: MEGAHIT {
-        ext.args   = { params.megahit_options ? params.megahit_options + "-m ${task.memory.toBytes()}" : "-m ${task.memory.toBytes()}" }
+        ext.args   = { params.megahit_options ? params.megahit_options + " -m ${task.memory.toBytes()}" : " -m ${task.memory.toBytes()}" }
```
Comment (d4straub) [inline conf/modules.config:260]:
```suggestion
        ext.args   = { params.megahit_options ? params.megahit_options + " -m ${task.memory.toBytes()}" : "-m ${task.memory.toBytes()}" }
```
the second white space isnt required I think, but might not hurt either.
Whats the default of megahit_options? If it is `""`, one could also do 
`{ params.megahit_options + " -m ${task.memory.toBytes()}" }` ?

**Review by d4straub — APPROVED** [review]
LGTM

File: conf/modules.config (hunk 2 above)
Comment (jfy133) [reply conf/modules.config:260]: OK yeah, lets do that

Comment (jfy133) [reply conf/modules.config]: There are no defaults, so it should be fine to inject `-m` as is

Comment (jfy133) [reply conf/modules.config]: Internet terrible here, github interface not working, so did it manually in https://github.com/nf-core/mag/pull/710/commits/52ccb1cdf4e4d4861a4521b51da3c6f6acec84ba

**Review by itrujnara — APPROVED** [review]
This is definitely one of the patches ever created. Looks good to me.

**Review by matthdsm — APPROVED** [review]

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.2.1 [2024-10-29]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## 3.2.1 [2024-10-30]
```

**Pull request merged**